tailwind.config.js

```diff
@@ -180,6 +180,36 @@ module.exports = {
         '1xl': '1.375rem'
       }
     },
+    height: {
+      '0': '0',
+      '1': '0.25rem',
+      '2': '0.5rem',
+      '3': '0.75rem',
+      '4': '1rem',
+      '5': '1.25rem',
+      '6': '1.5rem',
+      '8': '2rem',
+      '10': ' 2.5rem',
+      '12': ' 3rem',
+      '16': ' 4rem',
+      '20': ' 5rem',
+      '24': ' 6rem',
+      '30': ' 7.5rem',
+      '32': ' 8rem',
+      '40': ' 10rem',
+      '48': ' 12rem',
+      '56': ' 14rem',
+      '64': ' 16rem',
+      '128': ' 32rem',
+      auto: 'auto',
+      full: '100%',
```

Comment:
🚩  You should only need to include your added screen options here, rather than including options that we already get. We do it for maxHeight and maxWidth because there's not many options for those out of the box https://tailwindcss.com/docs/min-height#app 

As long as its in `extend` you're good just to add the missing ones 👍